gatsby-config.js

```diff
@@ -99,6 +99,7 @@ module.exports = {
               linkImagesToOriginal: false,
             },
           },
+          `gatsby-remark-copy-linked-files`,
```

Comment:
I had a problem where it wouldn't let me use gifs as images. This plugin solves this.